### PR TITLE
1. Setup

### PR DESCRIPTION
**Pull request by Kingjosh007**
I created a new React app using create-react-app.
To do so, I followed the instructions given in [this page from the official documentation](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
I used the correct Linters settings, for React & Redux.
I also wrote a professional README file.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```
Comment (Dipeshtwis) [inline .vscode/settings.json:3]: - [ ] I would not recommend pushing the `.vscode` file or folder into Github. instead, you can put that into the `.gitignore` file as these are unnecessary files which makes your project structure bad. Please remove it from Github.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Kingjosh007, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json (hunk 1 above)
Comment (Kingjosh007) [reply .vscode/settings.json]: Done.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @Kingjosh007, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**